src/components/TextInput/TextInput.styles.css.ts

```diff
@@ -0,0 +1,61 @@
+import { style } from '@vanilla-extract/css';
+
+const COLOR_ACTIVE = '#2C98FC';
+
+const textInputContainer = style({
+  position: 'relative',
+  border: '1px solid grey',
+  borderRadius: 6,
+  width: '100%',
+  padding: 10,
+  selectors: {
+    [`&:has(input:focus)`]: {
+      borderColor: COLOR_ACTIVE,
+    },
+  },
+});
+
+const cancelButton = style({
+  /* @ts-ignore-line */
+  position: 'absolute !important',
+  right: 0,
+  top: '2px',
+});
+
+const input = style({
+  background: 'none',
+  backgroundColor: 'white',
```

Comment:
I added white to palette as `theme.palette.neutral.base`, you can use it for all 'white' colors.